packages/syft/tests/syft/core/tensor/adp/gamma_tensor_test.py

```diff
@@ -10,6 +10,7 @@
 from syft.core.adp.data_subject_ledger import DataSubjectLedger
 from syft.core.adp.ledger_store import DictLedgerStore
 from syft.core.tensor.autodp.gamma_tensor import GammaTensor
+from syft.core.tensor.autodp.gamma_tensor import jax2numpy
 from syft.core.tensor.autodp.phi_tensor import PhiTensor as PT
 
```

Comment:
Could we add more tests here to make sure that lazyrepeatarray minvals/maxvals were initialized properly when going from PhiTensor -> GammaTensor, and that evaluate still works when you're adding GammaTensors?